client/src/Components/Signup/index.jsx

```diff
@@ -1,11 +1,84 @@
 import React from 'react';
+import './style.css';
+import { Form, Button } from 'react-bootstrap';
 import UnderConstruction from '../UnderConstruction';
 
 export default function SignUp() {
```

Comment:
this must be a class comp. not a function comp. because we need to store user info. on the state to send it to the backend
```suggestion
export default class SignUp extends React.Component {
```